websites/A/atomicradio/presence.ts

```diff
@@ -39,6 +110,27 @@ function clearPresenceData() {
 }
 
 presence.on("UpdateData", async () => {
+  if (window.location.origin === "https://status.atomicradio.eu") {
+    clearPresenceData();
+    presenceData.details = "Viewing the current status";
+    presenceData.startTimestamp = browsingTimestamp;
+    presence.setActivity(presenceData, true);
+    return;
+  }
+
+  if (window.location.origin === "https://docs.atomicradio.eu") {
```

Comment:
Instead of checking origin you could just check subdomain by ``window.location.host.split('.')[0]`` it will be easier and cleaner.